### PR TITLE
Charlie's Jenkins + C# + Okta post

### PR DESCRIPTION
**Pull request by Quorralyne**
This blog post has the following features:

- [x] A GitHub Repository with a polished README
- [x] A title that's approved by Dev Marketing
- [x] A URL that's approved by Dev Marketing
- [x] Rendered locally and confirmed that no Markdown typos exist

https://github.com/Okta-Bloggers/devblog/issues/161
https://docs.google.com/document/d/1pkMFFa2aPJs5L9B9KJtRGJv3l3QzFEiS2TH1QG3MhJk/edit#

### CODE REVIEW

File: _source/_posts/2019-07-26-jenkins-continuous-integration-csharp-aspnetcore.md [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+>* [ASP.NET Core SDK](https://dotnet.microsoft.com/download) and a suitable IDE (such as [VS Code](https://code.visualstudio.com/) or [Visual Studio](https://visualstudio.microsoft.com/vs/))
+>* [Jenkins installation](https://jenkins.io/download/) - choose the Long-term Support version for your platform
+
+
+## Install the Jenkins Continuous Integration Server
+Jenkins is a Java-based application and as such can run on any platform that has the appropriate Java runtime. There are also installer packages for many major operating systems as well as virtual machine templates that you can deploy to your favourite cloud provider. The steps required will vary depending on your operating system, and you can find specific details at https://jenkins.io/doc/book/installing/
+
+For the purposes of this walkthrough, we'll assume you're installing on Windows 10.
+
+Download the latest LTS version of Jenkins for Windows from https://jenkins.io/download/
```
Comment (reverentgeek) [inline _source/_posts/2019-07-26-jenkins-continuous-integration-csharp-aspnetcore.md]: All these "bare" links need to be wrapped in back-ticks [`]

File: _source/_posts/2019-07-26-jenkins-continuous-integration-csharp-aspnetcore.md [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+With that added, you're ready to commit your empty project:
+```sh
+git add -A
+git commit -a -m "Initial check-in"
+git push -u origin master
+```
+
+### Visual Studio 2017
+You can use the *New Project Wizard* in Visual Studio to create a new MVC application.
+Select File > New > Visual C# > .NET Core > ASP.NET Core Web Application
```
Comment (reverentgeek) [inline _source/_posts/2019-07-26-jenkins-continuous-integration-csharp-aspnetcore.md]: Here's another list of steps that need to be a numbered or bulleted list.

Comment (Quorralyne) [reply _source/_posts/2019-07-26-jenkins-continuous-integration-csharp-aspnetcore.md]: Gotcha.

File: _source/_posts/2019-07-26-jenkins-continuous-integration-csharp-aspnetcore.md (hunk 1 above)
Comment (Quorralyne) [reply _source/_posts/2019-07-26-jenkins-continuous-integration-csharp-aspnetcore.md]: Done.

[the remark this answers is not in the document]
Comment (Quorralyne) [reply _source/_posts/2019-07-26-jenkins-continuous-integration-csharp-aspnetcore.md]: Done.

**Pull request merged**